src/test/scala/BindingTests.scala

```diff
@@ -187,4 +187,65 @@ class BindingTests extends FunSuite {
   test("EntityRefMap arrays are equaly sized") {
     assert(EntityRefMap.keys.size == EntityRefMap.values.size)
   }
+
+  test("onClick = Function0") {
```

Comment:
Is there a way to test the following html

```html
<div class={myClass}>
  <input type="text"/>
</div>
```
and some interaction like this

- myClass is "blue"
- user types in "banana"
- myClass becomes "yellow"
- user input is still "banana"

The most important thing is that the user input is not removed by the class attribute update.